gamemodes/terrortown/gamemode/server/sv_propspec.lua

```diff
@@ -9,9 +9,7 @@ local timer = timer
 
 PROPSPEC = {}
 
----
--- @realm server
-local cvPropspecToggle = CreateConVar("ttt_spec_prop_control", "1", { FCVAR_NOTIFY, FCVAR_ARCHIVE })
+local cvPropspecToggle = GetConVar("ttt_spec_prop_control")
```

Comment:
just to be sure: did you test that the other file is loaded first?

Reply:
Yes, this works as expected.

It is loaded first:
cl main:
![image](https://github.com/user-attachments/assets/d8906876-7f73-4d84-b43d-732953c16bb4)

sv main:
![image](https://github.com/user-attachments/assets/b71439c0-e6c8-4bf2-addc-216c9d095314)